app/models/character.rb

```diff
@@ -26,7 +26,7 @@ class Character < ActiveRecord::Base
   nilify_blanks
 
   def icon
-    @icon ||= default_icon || galleries.detect(&:default_icon).try(:default_icon)
```

Comment:
We should just change uses of character.icon to character.default_icon but I'll merge for now.